packages/core/package.json

```diff
@@ -29,48 +10,49 @@
     "semver": "^7.1.3",
     "xcode": "^2.0.0"
   },
-  "devDependencies": {
-    "@babel/core": "^7.9.0",
-    "@babel/preset-env": "^7.9.5",
-    "@types/gzip-js": "^0.3.3",
-    "@types/jest": "^28.1.7",
-    "@types/node": "^12.7.2",
-    "@types/react-native": "^0.60.2",
-    "@types/semver": "^7.1.0",
-    "babel-jest": "^25.3.0",
-    "husky": "^4.2.5",
-    "jest": "^25.3.0",
-    "lerna": "^6.0.3",
-    "lint-staged": "^10.1.7",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-native": "^0.56.0",
-    "react-test-renderer": "^18.2.0",
-    "ts-jest": "^25.3.1",
-    "tslint": "^5.18.0",
-    "typedoc": "^0.17.8",
-    "typescript": "^4.7.4"
+  "peerDependencies": {
+    "react-native": ">=0.56.0"
   },
   "files": [
-    "lib",
-    "android",
-    "ios",
-    "RNEmbrace.podspec"
+    "lib"
   ],
-  "publishConfig": {
-    "access": "public"
-  },
-  "scripts": {
-    "tsc": "tsc",
-    "test": "node ./__tests__/state-tracker.test.ts",
-    "build": "yarn pack"
+  "author": "Embrace <support@embrace.io> (https://embrace.io/)",
+  "bugs": {
+    "url": "https://github.com/embrace-io/embrace-react-native-sdk/issues",
+    "email": "support@embrace.io"
   },
+  "homepage": "https://github.com/embrace-io/embrace-react-native-sdk",
+  "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": "git@github.com:embrace-io/embrace-io.git"
+    "url": "https://github.com/embrace-io/embrace-react-native-sdk",
+    "directory": "packages/core"
   },
-  "bugs": {
-    "url": "https://github.com/embrace-io/embrace-io/issues"
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "rnpm": {
+    "commands": {
+      "postlink": "node node_modules/embrace-io/dist/scripts/postlink",
```

Comment:
@TheMutsi were these postlink/unlink commands still needed? I wasn't sure if the path was correct if they were part of the core package I thought that would belong under node_moudles/@embrace-io/react-native/...

Reply:
No, we can delete it

Reply:
deleted